File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/inteligencia/perdidas/dto/ProductoVencidoDTO.java
package com.farmacia.sanidadbackend.inteligencia.perdidas.dto;

import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDate;

@Data
public class ProductoVencidoDTO {
    private Long loteId;
    private String numeroLote;
    private LocalDate fechaVencimiento;
    private Long medicamentoId;
    private String medicamentoNombre;
    private Integer cantidadVencida;
    private BigDecimal valorPerdido;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/ProductoVencidoDTO.kt
package com.sanidad.movil.data.remote.dto

data class ProductoVencidoDTO(
    val loteId: Long,
    val numeroLote: String,
    val fechaVencimiento: String,
    val medicamentoId: Long,
    val medicamentoNombre: String,
    val cantidadVencida: Int,
    val valorPerdido: Double
)